frontend/src/components/Forms/PartyRelationships/EditPartyRelationshipForm.js

```diff
@@ -21,13 +26,58 @@ const defaultProps = {
   mine: {},
 };
 
-const validate = (values) => {
+const checkDatesForOverlap = (values, props) => {
```

Comment:
If this is the same function as above can we make a single helper function?

Reply:
Please see the opening comment on this PR